PW.FluxQueryNet/FluxQueryBuilder.Aggregates.cs

```diff
@@ -0,0 +1,115 @@
+﻿using PW.FluxQueryNet.Extensions;
+
+namespace PW.FluxQueryNet
+{
+    public partial class FluxQueryBuilder
+    {
+        /// <inheritdoc/>
+        public IFluxStream Aggregate(string methodName, string? column = null)
+        {
+            _stringBuilder.AppendLine();
+            _stringBuilder.AppendPipe().Append(methodName).Append('(');
+
+            if (!string.IsNullOrWhiteSpace(column))
+                _stringBuilder.Append("column: \"").Append(column).Append('"');
+
+            _stringBuilder.Append(')');
+            return this;
+        }
+
+        /// <inheritdoc/>
+        public IFluxStream Sum(string? column = null) => Aggregate("sum", column);
+
+        /// <inheritdoc/>
+        public IFluxStream Count(string? column = null) => Aggregate("count", column);
+
+        /// <inheritdoc/>
+        public IFluxStream Mean(string? column = null) => Aggregate("mean", column);
+
+        /// <inheritdoc/>
+        public IFluxStream MovingAverage(int n)
+        {
+            _stringBuilder.AppendLine();
+            _stringBuilder.AppendPipe().Append("movingAverage(n: ").Append(n.ToFlux()).Append(')');
+            return this;
+        }
+
+        /// <inheritdoc/>
+        public IFluxStream TimedMovingAverage(FluxDuration every, FluxDuration period, string? column = null)
+        {
+            _stringBuilder.AppendLine();
+            _stringBuilder.AppendPipe().Append("timedMovingAverage(every: ").Append(every).Append(", period: ").Append(period);
+
+            if (!string.IsNullOrWhiteSpace(column))
+                _stringBuilder.Append(", column: \"").Append(column).Append('"');
+
+            _stringBuilder.Append(')');
+            return this;
+        }
+
+        /// <inheritdoc/>
+        public IFluxStream Mode(string? column = null) => Aggregate("mode", column);
+
+        /// <inheritdoc/>
+        public IFluxStream Spread(string? column = null) => Aggregate("spread", column);
+
+
+        /// <inheritdoc/>
+        public IFluxStream Window(FluxDuration every, FluxDuration? period = null, FluxDuration? offset = null,
```

Comment:
- `Window()` function still accept the `createEmpty` while it doesn't seem to work?